### DISCUSSION TITLE
retrieve feed list without access to opml export?

### DISCUSSION THREAD

**Discussion by luckybuzz** [FreshRSS/FreshRSS#7139] [Q&A]
Hi all, not a bug or feature request here, but hoping someone can help me.

I have been happily using FreshRSS for many months.  I know next to nothing about its setup or databases at all, but auto-installed FreshRSS on my host using Softaculous in cPanel.

This morning I clicked the button in Options to update to version 1.25.0.  I had updated that way once before, so figured it wouldn't be a problem.  However now I can't get FreshRSS to load in any browser, getting error 500.

Rather than trying to troubleshoot when I don't know anything about databases, php, etc.  I think I would like to just install an older version of FreshRSS again in Softaculous and start over again... start "fresh" as it were :P

Is there any way to retrieve a list of my subscribed feeds by accessing the data folders on my host server?  Without the ability to load FreshRSS in a browser, I can't use the export OPML option... but looking through these folders, I don't see anything that looks like it might contain a readable list of feed URLs.

Any help in finding and rescuing my list of feeds would be greatly appreciated.  Thanks!  iq.  :)

**Comment by Alkarex** [comment]
* First, try to find any error message in your logs - fixing the corresponding issue might be the easiest
* I do not know what database is used in your setup. Try to check if there is maybe a `db.sqlite` in your `FreshRSS/data/users/luckybuzz/` folder;
* Otherwise, if you have not exported your OPML feed list, you need to access your database to retrieve it, with something like `SELECT url FROM feed;`
* You can also try to revert to version 1.24.3

**Comment by math-GH** [comment]
Softaculous has some issues that we cannot investigate:

https://github.com/FreshRSS/FreshRSS/issues?q=is%3Aissue+Softaculous
